content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -2,77 +2,78 @@
 
-Every hour every lead without a score will be automatically scanned and
-assigned their right score according to your scoring rules.
+Every lead without a score is automatically be scanned every hour and is assigned the correct
+score according to your configured scoring rules.
 
-.. image:: media/lead_scoring03.png
+.. image:: lead_scoring/scoring-section-on-lead.png
    :align: center
+   :alt: Scoring Section on a Lead
 
 Assign leads
 ============
 
-Once the scores computed, leads can be assigned to specific teams using
-the same domain mechanism. To do so go to :menuselection:`CRM --> Leads Management --> Team Assignation`
-and apply a specific domain on each team. This domain can include scores.
+Once the score is computed, leads can be assigned to specific teams using the same domain
+mechanism. To do so, head to :menuselection:`CRM --> Leads --> Team Assignment` and apply a
+specific domain to each team. This domain can include scores.
 
-.. image:: media/lead_scoring04.png
+.. image:: lead_scoring/team-assignation.png
    :align: center
+   :alt: Team Assignments using Domains
 
-Further on, you can assign to a specific vendor in the team with an even
-more refined domain.
+If you want more granularity, you can assign leads to a specific salesperson on the team using
+further refined domains.
 
-To do so go to :menuselection:`CRM --> Leads Management --> Leads Assignation`.
+To do so go to :menuselection:`CRM --> Leads --> Leads Assignment`.
 
-.. image:: media/lead_scoring05.png
+.. image:: lead_scoring/lead-assignment-filters.png
    :align: center
+   :alt: Lead Assignments
 
 .. note::
-   The team & leads assignation will assign the unassigned leads
-   once a day.
+   The team & leads assignments are assigned to unassigned leads once per day.
 
 Evaluate & use the unassigned leads
 ===================================
 
-Once your scoring rules are in place you will most likely still have
-some unassigned leads. Some of them could still lead to an opportunity
-so it is useful to do something with them.
+Once your scoring rules are in place, you likely will still have some unassigned leads. Some of
+these leads could still transition into an opportunity, so ensuring they are handled is both
+useful and important.
 
-In your leads page you can place a filter to find your unassigned leads.
+On your leads overview page, you filter for your unassigned leads.
 
-.. image:: media/lead_scoring06.png
+.. image:: lead_scoring/unassigned-filter.png
    :align: center
+   :alt: Filtering for Unassigned Leads
 
-Why not using :menuselection:`Email Marketing` or
-:menuselection:`Marketing Automation` apps to send a mass email to
-them? You can also easily find such unassigned leads from there.
-
-.. image:: media/lead_scoring07.png
-   :align: center
+.. note::
+    You can also easily find unassigned leads by using the :menuselection:`Email Marketing` or
+    :menuselection:`Marketing Automation` apps to create a re-engagement campaign.
```

Comment:
You could use the [bold markup](https://www.odoo.com/documentation/14.0/contributing/documentation/rst_cheat_sheet.html#markup) instead of the menuselection directive, since the later is used to guide a user through a sequence of menus, which isn't the case here.